app/services/katello/pulp3/repository/apt.rb

```diff
@@ -0,0 +1,81 @@
+require 'pulp_deb_client'
+
+module Katello
+  module Pulp3
+    class Repository
+      class Apt < ::Katello::Pulp3::Repository
+        def remote_options
+          deb_remote_options = {
+            distributions: root.deb_releases
+          }
+          # TODO: need to define how to set these fields to empty
+          deb_remote_options[components] = root.deb_components if root.deb_components.present?
+          deb_remote_options[architectures] = root.deb_architectures if root.deb_architectures.present?
+
+          if root.url.blank?
+            deb_remote_options[:url] = nil
+          end
+
+          common_remote_options.merge(deb_remote_options)
+        end
+
+        def publication_options(repository_version)
+          super(repository_version).merge(
+            {
+              # FIXME: not yet implemented in the current pulp_deb version
+              #structured: true, # publish real suites (e.g. 'stable')
+              simple: true # publish all into 'default'-suite
```

Comment:
Is this still a TODO?  can eta on the pulp side?  Do we have to wait for that?

Reply:
Not sure, maybe @quba42 or @mdellweg can give feedback.
From what I understood, we do not have to wait for it.

Reply:
structured-mode is not necessary for subsciption-manager usage, so keeping the comment but removing the FIXME.